themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -50,7 +50,7 @@
 
     {block name='product_unit_price'}
       {if $displayUnitPrice}
-        <p class="product-unit-price">{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
+        <p class="product-unit-price"{hook h='displayProductPriceBlock' product=$product type="unit_price" returntype="style"}>{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
```

Comment:
Hi @paeddl,

Just to give me a little bit of context, in which case do you need to add style on this ?
Can't we imagine adding a CSS file to add this extra style when required ?

Thanks !